### PR TITLE
Status & Name, quick profile change in drawer (#294), length limit (#324)

### LINKED ISSUE #294: Status & Name, quick profile change in drawer
quick-profile-change

- [x] Instead of having to go to the profile to change name and status, you should also be able to do it quickly within the drawer.
- [x] tapping on the icon should open up profile page

### PR DESCRIPTION
**Pull request by alwx**
closes #294

### CODE REVIEW

**Comment by rasom** [comment]
Maybe better to use some placeholder here, because it is not obvious that there is an input.

**Comment by rasom** [comment]
When a new line begins the previous one is moved up a bit. @jarradh, is this a problem? 

prof-ed

**Comment by jarradh** [comment]
mmm it's not ideal, if its easy to fix then lets do it otherwise lets worry about it later, it's not exactly a dealbreaker

**Comment by rasom** [comment]
It would be better to submit the status when this button is pressed. Otherwise, user could be a bit confused how to save it

**Comment by rasom** [comment]
"Switch user" is a bit not in place

File: src/status_im/components/drawer/view.cljs [hunk 1]
```diff
@@ -37,47 +43,65 @@
...
+        (label :t/switch-users)]]]]]])
 
 (defn drawer-view [items]
-  [drawer-layout {:drawerWidth            260
-                  :render-navigation-view #(r/as-element [drawer-menu])
-                  :ref                    (fn [drawer]
-                                            (reset! drawer-atom drawer))}
+  [drawer-layout {:drawerWidth          260
+                  :renderNavigationView #(r/as-element [drawer-menu])
+                  :onDrawerSlide        #(dismiss-keyboard!)
```
Comment (rasom) [inline src/status_im/components/drawer/view.cljs]: Maybe just `dismiss-keyboard!` will be enough?

**Comment by alwx** [comment]
> It would be better to submit the status when this button is pressed. Otherwise, user could be a bit confused how to save it 

@rasom 
unfortunately, we can't simply do it when multiline = true. The similar bug happens when you try to send a message in chat by pressing Enter.

my suggestion is to move forward and fix all those bugs in another task (#322)

**Comment by rasom** [comment]
Seems that on iOS `status` input is not centered horizontally and has different value for `margin-top`:

**Pull request merged**